tools/oci-cli-wrapper/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+//! ImageTool enablement library implements a standardized way of calling commandline container image
+//! tools for interacting primarily with kit images in a container registry.
+//!
+//! Current two tools are supported:
+//! * crane, gcrane, kcrane
+//!     Crane provides a more direct interaction with the container registry, allowing us to query image information in the registry without
+//!     having to pull the full image to disk. It also does not require a daemon to operate and has optimizations for pulling large images to disk
+//! * docker
```

Comment:
nit: no love for Docker? we could mention that the containerd snapshotter must be enabled, or talk about why it's less efficient (save/load cycle, have to pull before inspecting, etc)